app/src/main/res/values/dimens.xml

```diff
@@ -35,6 +35,7 @@
     <dimen name="toolbar_elevation">12dp</dimen>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/c7380d74fc32b4ddceef02c8d9a561008c986f4f/app/src/main/res/values/dimens.xml#L35"></span>The resource <code>R.dimen.toolbar_elevation</code> appears to be unused</td>
    </tr>
  </tbody>
</table>